src/activity_handlers.cpp

```diff
@@ -524,25 +524,35 @@ void butchery_drops_harvest( const mtype &mt, player &p, int age, const std::fun
         const itype *drop = item::find_type( entry.drop );
 
         if( roll <= 0 ) {
-            p.add_msg_if_player( m_bad, _( "You fail to harvest: %s" ), drop->nname( 1 ).c_str() );
+            auto msg = mt.species.count( species_id( "ROBOT" ) ) ?
```

Comment:
Yes, I agree but the problem is that currently we handle robot corpses via:

1. Two monster death functions `BROKEN` and `BROKEN_AMMO`...
2. ...which stringify the name to an item, eg. `mon_foo` to `broken_foo`
2. Then a separate item definition for each `broken_foo`
4. An uncrafting recipe for each `broken_foo`

I'm open to (concise) suggestions as to alternatives but don't especially want to hold up the PR for them.